common-apps-functions-source.sh

```diff
@@ -1962,6 +1962,8 @@ function build_cross_binutils()
           config_options+=("--with-pkgversion=${BRANDING}")
           config_options+=("--with-system-zlib")
 
+          config_options+=("--with-sysroot=${APP_PREFIX}/${GCC_TARGET}")
```

Comment:
I don't see `--with-sysroot` in the list of available options for binutils. Where does it come from?

Reply:
yes, you are right. it's unnecessary, my bad.